### PR TITLE
dev/report#5 - Fix mailing report unique count issue

### PR DESCRIPTION
**Pull request by sunilpawar**
Overview
----------------------------------------
Mailing -> Scheduled and Sent Mailings -> Report
Unique Opens  -> Report
Total Opens   -> Report
Before
----------------------------------------
Both Report showing same count (Unique report is in-correct)

After
----------------------------------------
Unique Opens Report has less value as compared to Total Opens

### CODE REVIEW

**Comment by eileenmcnaughton** [comment]
If we are meddling with group by on a report we should include a test - check api_v3_ReportTemplateTest for a starting point

**Comment by eileenmcnaughton** [comment]
@yashodha seems there are a few CiviDesk PRs incoming - we should figure out a plan so they don't stall on lack of tests etc & so we can organise some review trading so they get reviewed

**Comment by Stoob** [comment]
Perhaps not related, but I found that the Opened.php report wasn't functional.  

https://github.com/civicrm/civicrm-core/pull/12432

**Comment by eileenmcnaughton** [comment]
@sunilpawar @yashodha are you planning on following through on getting these PRs to a reviewable point - ie. adding tests (otherwise we should close & track through gitlab)

**Comment by sunilpawar** [comment]
@eileenmcnaughton, I have added test case, can you confirm the test-case is ok?

File: tests/phpunit/api/v3/ReportTemplateTest.php [hunk 1]
```diff
@@ -139,6 +139,19 @@ public function testReportTemplateGetRowsContactSummary() {
      */
   }
 
+  /**
+   * Test getrows on Mailing Opened report.
+   */
+  public function testReportTemplateGetRowsMailingUniqueOpened() {
+    $description = "Retrieve rows from a mailing opened report template.";
+    $result = $this->callAPIAndDocument('report_template', 'getrows', array(
```
Comment (eileenmcnaughton) [inline tests/phpunit/api/v3/ReportTemplateTest.php]: Just use callApiSuccess unless you are clear that you want to create an example (& if you do that you should check it in)

**Comment by eileenmcnaughton** [comment]
@sunilpawar can you check the test

**Comment by monishdeb** [comment]
Jenkins test this please

**Comment by monishdeb** [comment]
Closing in favor of https://github.com/civicrm/civicrm-core/pull/13322

**Pull request closed without merging**